drivers/pds/lib/utils.go

```diff
@@ -735,8 +749,8 @@ func ValidatePDSDeploymentStatus(deployment *pds.ModelsDeployment, healthStatus
 			return false, err
 		}
 		if !strings.Contains(status.GetHealth(), healthStatus) {
-			err = fmt.Errorf("status %v", status.GetHealth())
-			return false, err
+			log.Errorf("status: %v", status.GetHealth())
+			return false, nil
```

Comment:
Why are you logging it as error if you not returning it? Either return it as verbose error, don't log it as error or elaborate why this is an error and why you are not returning it as an error

Reply:
done